pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -544,6 +544,9 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 
 	// Create kubevirt VM from source VM:
 	reqLogger.Info("Creating a new VM", "VM.Namespace", vmSpec.Namespace, "VM.Name", vmSpec.Name)
+
+	vmJSON, _ := json.Marshal(vmSpec)
+	reqLogger.Info("VM struct", "VM spec", string(vmJSON))
```

Comment:
It can be a lot of text. Should it be here?

Reply:
There is a bit of text but I found it useful more than once.

Reply:
I'm in favor of logging for debug / error handling, especially when we failed to create the VM - we won't be able to debug the VM request easily.

However, how about moving this inside the condition - since if we managed to create the VM, we'll have the VM resource created on etcd to which we can refer for debug.
The main value as I see it if we failed to create the VM.

Also, perhaps yaml format ?

Reply:
Good idea, will move. Json takes one line - no spaces or formatting.